src/write.rs

```diff
@@ -431,14 +431,30 @@ where
     /// Remove the given value from the value-bag of the given key.
     ///
     /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    #[deprecated(since = "11.0.0", note = "Renamed to remove_value")]
     pub fn remove(&mut self, k: K, v: V) -> &mut Self {
+        self.remove_value(k, v)
```

Comment:
Is this the right way to deprecate this? So that if people are using the old one, they get new functionality? Not sure if that's good or not. Also not sure if this function should have its documentation removed so it doesn't get out of sync if `remove_value` changes? Or maybe I just shouldn't change the innards of `remove`?

Reply:
There's no new functionality here though, right? Since it's just a rename, I think it's fine.

Reply:
Not yet! I was just thinking that if someone goes and updates `remove_value` for some reason, should those still using `remove` get that new change?

Reply:
If `remove_value` gets updated without its semantics change, then yes, I think `remove` should also get that change. And changing the semantics of `remove_value` would be backwards-incompatbile anyway. So I think it's fine.